software-packages/gromacs.rst

```diff
@@ -32,117 +32,70 @@ Running parallel GROMACS jobs: pure MPI
 GROMACS can exploit multiple nodes on Cirrus and will generally be run in
 exclusive mode over more than one node.
 
-For example, the following script will run a GROMACS MD job using 4 nodes
-(144 cores) with pure MPI.
+For example, the following script will run a GROMACS MD job using 2 nodes
+(72 cores) with pure MPI.
 
 ::
 
    #!/bin/bash --login
    
-   # PBS job options (name, compute nodes, job time)
-   #PBS -N mdrun_test
-   #PBS -l select=4:ncpus=36
+   # Slurm job options (name, compute nodes, job time)
+   #SBATCH --job-name=gmx_test
+   #SBATCH --nodes=2
+   #SBATCH --tasks-per-node=36
+   #SBATCH --time=0:25:0
    # Make sure you are not sharing nodes with other users
-   #PBS -l place=scatter:excl
-   #PBS -l walltime=0:20:0
+   #SBATCH --exclusive
    
    # Replace [budget code] below with your project code (e.g. t01)
-   #PBS -A [budget code]
-   
-   # Change to the directory that the job was submitted from
-   cd $PBS_O_WORKDIR
+   #SBATCH --account=[budget code]
    
    # Load GROMACS and MPI modules
    module load gromacs
-   module load mpt
 
    # Run using input in test_calc.tpr
    # Note: '-ppn 36' is required to use all physical cores across
    # nodes as hyperthreading is enabled by default
    OMP_NUM_THREADS=1 
-   mpiexec_mpt -ppn 36 -n 144 gmx_mpi mdrun -s test_calc.tpr
+   srun -n 72 gmx_mpi mdrun -s test_calc.tpr
```

Comment:
@jsindt Thanks for the changes. Please remove the `-n 72` option from srun as it is spurious.